packages/tools/devtools/devtools-view/src/test/TelemetryView.test.tsx

```diff
@@ -0,0 +1,90 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+	type DevtoolsFeatureFlags,
+	DevtoolsFeatures,
+} from "@fluidframework/devtools-core/internal";
+// Normal usage pattern for @testing-library/jest-dom
+// eslint-disable-next-line import/no-unassigned-import
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+import { userEvent } from "@testing-library/user-event";
+import React from "react";
+// eslint-disable-next-line import/no-internal-modules
+import { act } from "react-dom/test-utils";
+
+import { MessageRelayContext } from "../MessageRelayContext.js";
+import { TelemetryView } from "../components/index.js";
+
+import { assertNoAccessibilityViolations, MockMessageRelay } from "./utils/index.js";
+
+describe("TelemetryView Accessibility Check", () => {
+	const supportedFeatures: DevtoolsFeatureFlags = {
+		telemetry: true,
```

Comment:
Should we test the accessibility when `telemetry` is false?

Reply:
We could, but I don't think that's necessary. Do we have use cases where telemetry is false?

Reply:
Yeah, if a user doesn't provide the devtools with a `DevtoolsLogger` at setup, then this flag will be false.

Reply:
As discussed, the TelemetryView doesn't query for supportedFeatures. If telemetry is false, the Events tab  and TelemetryView won't be rendered